examples/deck/example.js

```diff
@@ -203,6 +215,82 @@ export default class Example extends Component<
     );
   }
 
+  getLayerOverrides() {
+    // get geometry layer
+    const layerOverrides =
+      this.state.geometryType === 'outlined-geojson'
+        ? {
+            ['geojson']: {
```

Comment:
Is the override done by layer `id`? Interesting, I can see how this can be useful. I always thought it would be keyed on layer type, but of course that gets tricky in deep nesting. Maybe support both?

Reply:
A composite layer can compose multiple layers of the same type. So I think `id` is the unique way of identifying sub layers to override.